compiler/qsc_project/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+//! This module handles the logic that constitutes the Q# project system.
```

Comment:
This and other new files must have the boilerplate Microsoft copyright comment at the top, ie:
```rust
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
```